playbooks/support/responding.md

```diff
@@ -75,28 +93,85 @@ One of the on-call members handles internal and external communication for the i
 
 ## Step 5: Resolve
 
 Incidents are resolved when the immediate issue has been mitigated. After that, the team needs to make sure all
-follow-up items are tracked and the Jira Ops ticket clearly reflects this state.
+follow-up items are tracked and the OpsGenie incident clearly reflects this state.
 
 1. Create any necessary follow-up tickets either in the bug backlog or a specific team’s project board. Link these
-   from within the Jira Ops ticket.
-2. Post any final updates to the Jira Ops ticket
+   from within the OpsGenie incident.
+2. Post any final updates to the OpsGenie incident
 
-- Update timestamps
-- Mark the ticket as **RESOLVED**
-- Add any additional labels or metadata that may be relevant later
+   - Add a timeline entry describing why you're resolving the incident
+   - Mark the incident as **RESOLVED**
 
-3. Resolve any outstanding incidents on our status page via the integration on the Jira Ops ticket.
-4. If the incident was Critical, create a post mortem following the guidelines in our
-   [post_mortems repo](https://github.com/artsy/post_mortems).
+3. Resolve any outstanding incidents on our status page via statuspage.io.
+4. If the incident was Critical, create a post mortem in OpsGenie.
 5. Update relevant [playbooks](https://github.com/artsy/potential/wiki) 🔒 with any lessons.
+
+## Severity of Incidents
+
+### P1 - **Critical**
+
+Critical incidents have a very high impact:
+
+- A user-facing service is down for all users.
+- Confidentiality or privacy is breached.
+- Customer data loss.
+- “All hands on deck.”
+
+#### Examples
+
+- Artsy website is down.
+- Artsy website is under DDoS attack.
+
+### P2 - **High**
+
+High incidents have a significant impact:
+
+- A user-facing service is down for a subset of users.
+- Core functionality is impacted for a specific use-case.
+- Need to find the best people to solve as soon as possible.
+
+#### Examples
+
+- Partners unable to delete artworks in CMS
+- Multiple partners unable to add credit cards
+- Artwork pages showing duplicate information
+
+### P3 - **Moderate**
+
+Moderate incidents have a low impact but require some urgency:
+
+- A user-facing service is down for a single user.
+- Core functionality is impacted for a single resource.
+- Should inform the relevant product team about the issue.
+
+#### Examples
+
+- Broken links in Eigen on UNTITLED art fair page
+- A single partner cannot access CMS
+- User unable to place bid on lot in live sale, needs to be placed in the backend
+
+### P4 - **Low**
+
+Low incidents have low impact and low urgency:
+
+- Known issues that are on a product team’s radar.
+- Issues that can be handled in a regular sprint’s work.
+
+#### Examples
+
+- Charge details for a partner need updating.
+- Artwork appearance on the site (“how come it’s not showing up in…”).
+
+### P5 - **Informational**
+
+Informational incidents are used primarily for training. Use this for incidents you create to practice incident
+response.
```

Comment:
I wonder if we can be a bit less specific with this severity level.

I found this definition from Norton:

> These are routine security incidents with no impact to the client. They are presented for informational/reporting purposes.

source: https://us.norton.com/online-threats/glossary/s/security-incident-severity--informational.html

Maybe we can use something like:

Informational incidents have no customer impact. They are often created for reporting or training purposes.

Reply:
Good call, I like it. Changed!